src/som/interpreter/actors/EventualMessage.java

```diff
@@ -22,15 +22,25 @@
   protected final RootCallTarget onReceive;
   protected final EventualMessage causalMessage;
 
-  protected final boolean triggerRcvrBreakpoint;
+  /**
+    * It is not final because its value can be updated in order that other breakpoints
+    * can reuse the stepping strategy implemented for message receiver breakpoints.
+    */
+  protected boolean triggerRcvrBreakpoint;
```

Comment:
triggerRcvrBreakpoint should be named `triggerMessageReceiverBreakpoint` to be unambiguous within the new set of breakpoints